init.sh

```diff
@@ -22,17 +22,19 @@ done
 if test -z "$machine" ; then
    echo "An invalid target name was given of $choice, available targets are:"
    printf '%s\n' "${targets[@]}"
-   unset machine choice
+   unset machine choice eula
+   echo "Usage: source init.sh target [accept-eula] [-f]"
```

Comment:
Yes I suppose that is valid, as it stands I've always presumed first is machine second is eula. I only raised it because the eula is only relevant to dragonboard and -f is only relevant to non qemu (or going forward 'non-supported' machines). This would be easier to implement with user interactions during execution, but then it make it harder to automate.